R/input_output_filter.R

```diff
@@ -19,12 +19,16 @@
 #'  value) you would like to use to remove samples from the imported data.
 #' @param keep_vals Alternatively, keep only samples represented by these 
 #'  values.
-#' @return A list variable with (1) the loaded taxa table, and (2) the loaded 
-#'  mapping file.
+#' @return A list variable with (1) the loaded data, (2) the loaded mapping
+#'  file, and (3) (if .biom format) the loaded taxonomies
 #' @examples 
 #' \dontrun{
-#' load_taxa_table("filepath_to_OTU_table.txt", "filepath_to_mapping_file.txt",
-#'   "sample_type", filter_vals = "blank")
+#' tax_table_fp = system.file('extdata', 'fruits_veggies_taxa_table_wTax.biom', 
```

Comment:
Let's keep these examples the way they were - I realize they don't run as is, but this is how users will actually be using the commands. I worry that including the system.file functions will confuse people